bothan-lib/src/telemetry/metrics.rs

```diff
@@ -0,0 +1,61 @@
+use opentelemetry::{global, KeyValue};
+
+pub struct Metrics{}
```

Comment:
the metrics seems specific to each type, maybe splitting it into 2 `Metrics` one for ws and one for rest would be better